source/parameters/bend.md

```diff
@@ -0,0 +1,193 @@
+  ```
+  See the discussion of `g` and `Kn0` below for more details.
+%
+- **bend_field (output param), norm_bend_field (output_param)**
+The actual dipole bend field ignoring any skew field component which is set by `Bs0`.
+The relation between this and `bend_field_ref` is
+  ```{code} yaml
+  bend_field = bend_field_ref + Bn0 * cos(tilt0) + Bs0 * sin(tilt0)
+  ```
+%
+- **bend_type**
+The `bend_type` parameter sets the "logical shape" of the bend.
+This parameter can take values of
+  ```{code} yaml
+  RECTANGULAR  - or
+  SECTOR       - the default
+  ```
+  The logical shape of a bend, in most situations, is irrelevant.
+  The only case where the logical shape is used is when the bend angle is varied.
+  In this case, for a `SECTOR` bend, the face angles `e1` and `e2` are
+  held constant and `e1_rect` and `e2_rect` are varied to keep the relationship
+  between `e1` and `e1_rect`, and `e2` and `e2_rect` satisfied as discussed below.
+%
+- **e1, e2**
+The values of `e1` and `e2` gives the rotation angle of the entrance and exit pole faces
+respectively with respect to the radial {math}`x_1` and {math}`x_2` axes as shown in {numref}`f:bend`.
+Zero `e1` and `e2` gives a wedge shaped magnet.
+Also see `e1_rect` and `e2_rect`. The relationship is
+  ```{code} python
+  :label: eeaeea
+
+  e1 = e1_rect + angle/2 
+  e2 = e2_rect + angle/2
+  ```
+%
+- **e1_rect, e2_rect**
+Face angle rotations like `e1` and `e2` except angles are measured with respect to
+fiducial lines that are parallel to each other and rotated by `angle`/2 from the radial
+{math}`x_1` and {math}`x_2` axes as shown in {numref}`f:sbend`.
+Zero `e1_rect` and `e2_rect` gives a rectangular magnet shape.
+%
+- **edge_int1, edge_int2**
+The field integral for the entrance pole face `edge_int1` is given by
+  ```{math}
+  \text{edge_int1} = \int_{pole} \!\! ds \frac{B_y(s) (B_{y0} - B_y(s))}
+  {2 , B_{y0}^2}
+  ```
+  For the exit pole face there is a similar equation for `edge_int2`
+
+  Note: In Bmad and MAD, these integrals are represented by the product of `fint` and `hgap`.
+
+  `edge_int1` and `edge_int2` can be related to the Enge function which is sometimes used to model the
+  fringe field. The Enge function is of the form
+  ```{math}
+  B_y(s) = \frac{B_{y0}}{1 + \exp[P(s)]}
+  ```
+  ```{math}
+  P(s) = C_0 + C_1 , s + C_2 , s^2 + C_3 , s^3 + , \ldots
+  ```
+  The {math}`C_0` term simply shifts where the edge of the bend is. If all the {math}`C_n` are zero except for
+  {math}`C_0` and {math}`C_1` then
+  ```{math}
+  C_1 = \frac{1}{2 \cdot \text{field_int}}
+  ```
+%
+- **g, rho (output param)**
+The Reference bending radius which determines the reference coordinate system is `rho` (see
+[](#s:ref)). `g` = `1/rho` is the "bend strength" and is proportional to the Reference
+dipole magnetic field. `g` is related to the reference magnetic field `bend_field_ref` via
+  ```{math}
+  :label: gqpb
+
+  \text{g} = \frac{q}{p_0} \cdot \text{bend_field_ref}
+end{equation}
```

Comment:
```suggestion
\end{equation}
```

Reply:
This `end{equation}` appears as text in the rendered document.

Reply:
Fixed.